lib/core/engine/collector.js

```diff
@@ -168,8 +168,8 @@ class Collector {
               data.visualMetrics.FirstVisualChange
             } SpeedIndex: ${
               data.visualMetrics.SpeedIndex
-            } PerceptualSpeedIndex: ${
```

Comment:
This should be a separate commit.  Is upstream likely to take this change?  Would they take just _adding_ `VisualComplete85`?

Reply:
This isn't something we would uplift.  I think it's just more valuable for our purposes to display VC85 instead of PerceptualSpeedIndex on each iteration.  That being said, you're right and it probably should be in another commit.